README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img height="100" src="https://phylum.io/logo/dark-bckg.svg">
+  <img height="100" src="https://app.phylum.io/static/media/phylum-blue.d9e09fd4ac5774237f0c44f3173cb783.svg">
```

Comment:
The hash in the filename means we have to change this link anytime the logo changes....